src/inc/winrt/paraminstanceapi.h

```diff
@@ -419,7 +419,7 @@ namespace Ro { namespace detail {
         __in_ecount(size) const T* src, 
         size_t               size)
     {
-        for (size_t i = 0; i != _size; ++i) 
+        for (size_t i = 0; i != size; ++i) 
```

Comment:
You may want to check if this function is actually used. Since it used to compile there's a good change that it is not and it compiled only due to the way older versions of VC++ handle templates.

Reply:
@mikedn indeed it is not used, at least after removing it code still compiling. Is it safe to be removed? In the docs seems that code is compiled using VS 2015 too, but I only check using VS 2017 toolchain.

Reply:
I think it is ok to keep it for symmetry, even though it is unused right now.

Reply:
Right, it's not 100% unused as it may seem. This is template code so it's possible, at least in theory, that at some point someone will attempt to instantiate with a type parameter different from `char`/`wchar_t`